qa/L0_python_api/test.sh

```diff
@@ -51,7 +51,14 @@ fi
 
 
 FRONTEND_TEST_LOG="./python_kserve.log"
-python -m pytest --junitxml=test_kserve.xml test_kserve.py > $FRONTEND_TEST_LOG 2>&1
+# TODO: [DLIS-7215] Run tritonclient.grpc as separate process
```

Comment:
Is 7215 the right ticket? It has no description and is in-review

Reply:
Apply everywhere else you see 7215

Reply:
Right ticket is DLIS-7735. Thank you for the catch. Updated all references.